apps/semver-yeasy/semver-yeasy.sh

```
@@ -90,17 +109,23 @@ calculate-version)
     if [ "${repo_type}" = 'SINGLE_APP' ]; then
         service_versions_txt='## version bump\n'
         if [ "${SEMVERYEASY_CHANGED}" = 'true' ]; then
-        ${GITVERSION_EXEC_PATH} $(pwd) /config "${CONFIG_FILE}"
-        gitversion_calc=$(${GITVERSION_EXEC_PATH} $(pwd) /config "${CONFIG_FILE}")
+        CONFIG_FILE="${!CONFIG_FILE_VAR}"
+        ${GITVERSION_EXEC_PATH} $(pwd) /nonormalize /config "${CONFIG_FILE}"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)